goon/browserassets/css/browserOutput.css

```diff
@@ -313,6 +313,11 @@ h1.alert, h2.alert			{color: #000000;}
 .slime					{color: #0077AA;}
 .drask					{color: #a3d4eb; font-family: "Arial Black";}
 .clown					{color: #ff0000;}
+.grynn					{color: #838d6c;}
+.sohlee					{color: #7a8261;}
```

Comment:
That's because these are tab characters, and not spaces, so the width of each tab can appear different to other editors.